app/locales/en/translations.ts

```diff
@@ -1108,6 +1111,72 @@ export default {
             form_view: {
                 no_files: 'No files uploaded',
             },
+            withdraw: {
+                warning: 'Withdrawing a registration will remove its content from the OSF, but leave basic metadata behind. The title of a withdrawn registration and its contributor list will remain, as will justification or explanation of the withdrawal, should you wish to provide it. Withdrawn registrations will be marked with a "withdrawn" tag. This action is irreversible.',
+                withdraw: 'Withdraw registration',
+                withdrawal_justification_label: 'Please provide your justification for withdrawing this registration.',
+                random_scientist_x: 'Type <strong>{{x}}</strong> below and click Withdraw Registration if you are sure you want to continue.',
+                success: 'Withdrawal request submitted successfully',
+                error: 'Unable to withdraw this registration',
+            },
+            aria_labels: {
+                main: 'Toolbar: withdraw, fork, share registration',
+                scientist_name_input: 'Random scientist name',
+            },
+            embargoed: {
+                state: 'Embargoed',
+                action: 'End embargo early',
+                action_success: 'Embargo termination request successfully sent',
+                action_error: 'Unable to submit embargo termination request',
+                confirm_text: 'By clicking confirm, an email will be sent to project administrator(s) to approve ending the embargo. If approved, this registration, including any components, will be made public immmediately. This action is irreversible.',
+                confirm_button_text: 'Confirm',
+                banner: 'This registration is currently embargoed. It will remain private until its embargo end date, {{embargoEndDate}}.',
+            },
+            pending: {
+                state: 'Pending',
+                action: {
+                    approve: 'Approve',
+                    reject: 'Reject',
+                },
+            },
+            public: {
+                state: 'Public',
+            },
+            withdrawn: {
```

Comment:
With #577, withdrawn registrations get a tombstone instead of a banner, so we can probably drop this one.